system/ee/legacy/libraries/addons/Addons_installer.php

```diff
@@ -156,6 +156,26 @@ function install_fieldtype($fieldtype)
 
 			$default_settings = $FT->install();
 
+			if (!empty($FT->is_option_fieldtype)) {
```

Comment:
I think this does not account case where `is_option_fieldtype` is not set at all.
Plus probably should check for boolean and not just if empty